zaps/0002-encrypt-push-notifications.md

```diff
@@ -43,9 +43,7 @@ in between the individual Zulip server and its users.
 # Design
 
 Each mobile push notification a Zulip server sends will be encrypted
-with a public key belonging to the intended user device.
-The corresponding private key will have been generated by the Zulip
-mobile app on the device, and will remain only on the device.
+with a symmetric key generated by the client and shared with the server.
```

Comment:
The term "client" gets defined a couple of paragraphs later. In general, at this early stage of the intro, the text is more explicit:

```suggestion
with a secret key specific to the intended user device.
The key will have been generated by the Zulip mobile app
on the device, and shared only between that device
and the server.
```